src/main/java/com/faforever/client/preferences/ChatPrefs.java

```diff
@@ -27,6 +29,7 @@
   private final IntegerProperty channelTabScrollPaneWidth;
   private final MapProperty<String, Color> userToColor;
   private final BooleanProperty hideFoeMessages;
+  private final StringProperty ukTime;
```

Comment:
This should be an ObjectProperty holding `TimeInfo`, so we can avoid manual forth-and-back string conversions and string comparisons.